content/en/docs/concepts/signals/traces.md

```diff
@@ -90,106 +93,111 @@ Sample Trace:
     ],
 }
 ```
-
-To understand how tracing in OpenTelemetry works, let's look at a list of
-components that will play a part in instrumenting our code:
-
+ 
+A key Trace component in OpenTelemetry is a Span:
+ 
+A [**Span**](/docs/concepts/observability-primer/#spans) represents a unit of
+work or operation. Spans are the building blocks of Traces.
+ 
+We'll dive into Spans in [greater
+detail](/docs/concepts/observability-primer/#spans), but let's first look at the
+list of OpenTelemetry components that play a part in instrumenting our code:
+ 
 - Tracer
 - Tracer Provider
 - Trace Exporter
 - Trace Context
-
-### Tracer Provider
-
-A Tracer Provider (sometimes called `TracerProvider`) is a factory for
-`Tracer`s. In most applications, a Tracer Provider is initialized once and its
-lifecycle matches the application's lifecycle. Tracer Provider initialization
-also includes Resource and Exporter initialization. It is typically the first
-step in tracing with OpenTelemetry. In some language SDKs, a global Tracer
-Provider is already initialized for you.
-
+ 
 ### Tracer
-
-A Tracer creates spans containing more information about what is happening for a
-given operation, such as a request in a service. Tracers are created from Tracer
-Providers. In some languages, a global Tracer is already initialized for you.
-
+ 
+A `Tracer` creates spans, where each span has information about what's happening
+for a given operation; for example a request in a service. Tracers are created
+from `TracerProvider`s. In some languages, a global Tracer is already
+initialized for you.
+ 
+### Tracer Provider
+ 
+A **Tracer Provider** (sometimes called `TracerProvider`) is a factory to create
+Tracers. In most applications, a Tracer Provider is initialized once, and the
+Trace Provider's lifecycle matches the application's lifecycle.
```

Comment:
```suggestion
A **tracer provider** is a factory used to create tracers. 
In most applications, a tracer provider is initialized once, and the
tracer provider's lifecycle matches the application's lifecycle.
```

Reply:
How do we distinguish the descriptive words (eg "tracer provider") vs the actual object name?